data/mllib/sample_isotonic_regression_data.csv

```diff
@@ -0,0 +1,101 @@
+4710.28,500.00,1.00
```

Comment:
Shall we make the data in range [0, 1] (to simulate predictions from logistic regression)? If the weights are always `1.0`, we can remove it from the sample data.